pkg/cli/term/setup_unix.go

```diff
@@ -41,12 +39,12 @@ func setup(in, out *os.File) (func() error, error) {
 	}
 
 	var errFlushInput error
-	if flushInputDuringSetup {
+	/*
 		err = sys.FlushInput(fd)
 		if err != nil {
 			errFlushInput = fmt.Errorf("can't flush input: %s", err)
 		}
-	}
+	*/
```

Comment:
Don't comment out code, just remove it.

Reply:
Fine by me. I only made the implicit commenting out explicit because it wasn't clear why that code was never executed. As in, you didn't include a comment as to why the code was hidden behind `if false`. :smile: So I opted to keep it as comments just in case you intended to enable it at some point in the future.